client/app/shared/modals/data-upload/data-upload.component.ts

```diff
@@ -0,0 +1,127 @@
+import {Component} from '@angular/core';
+import {UploadEvent, UploadFile, FileSystemFileEntry} from 'ngx-file-drop';
+
+import {ParserService} from '../../services/parser.service';
+import {TournamentDataService} from '../../services/tournament-data.service';
+import {TeamsService} from '../../services/teams.service';
+import {Team} from '../../models/team';
+import {Notifications} from '../../services/notifications.service';
+
+@Component({
+    selector: 'data-upload',
+    templateUrl: './data-upload.component.html',
+    styleUrls: ['./data-upload.component.css']
+})
+export class DataUpload {
+    public file: UploadFile
+    public content: string
+    public fileHovering: Boolean
+    public loading: Boolean
+    public data: any
+    public teams: Array<Team>
+    public isSchedule: Boolean
```

Comment:
I think  a better idea is to save the file type, i.e. schedule/team list so if we will add another file type it will be easy to implement